crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -74,6 +74,9 @@ pub trait FileSystem: ValueToString {
     fn read(&self, fs_path: FileSystemPathVc) -> FileContentVc;
     fn read_link(&self, fs_path: FileSystemPathVc) -> LinkContentVc;
     fn read_dir(&self, fs_path: FileSystemPathVc) -> DirectoryContentVc;
+    fn track(&self, _fs_path: FileSystemPathVc) -> CompletionVc {
+        CompletionVc::immutable()
```

Comment:
The default impl can call `read()` to track it.

Reply:
This really need to be changed otherwise the default impl behaves incorrectly

Reply:
Done.

Reply:
Actually, I'm removing this default impl entirely. The default gets us back into doubling the memory again (write -> track -> read means we have a write content and a read content in memory).